src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,89 @@
 package core.basesyntax;
 
+import java.util.LinkedList;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT = 1 << 4;
+    private static final float LOAD_FACTOR = 0.75f;
+    private LinkedList<Entry<K,V>>[] buckets;
```

Comment:
The field 'null1' is not following Java naming conventions and its purpose is unclear. Consider renaming it to something more informative, like 'nullKeyMappedValue', to reflect its usage as the value for the null key.